packages/accounts-base/accounts_client.js

```diff
@@ -443,7 +449,7 @@ export class AccountsClient extends AccountsCommon {
   // before callbacks are registered see #10157
   _startupCallback(callback) {
     // Are we already logged in?
-    if (this.connection._userId) {
+    if (this._loginCallbacks_called) {
```

Comment:
@renanccastro,
This scenario is the essence of the original bug #10157. 
> - _startupCallback is called and the login was done before it (`onLogin` callback is never called)

#10543 added this code to solve it and has been in the code since `1.8.2`.  

This PR addresses the problem that `this.connection._userId` being set does not imply the user is fully logged in and the loginCallbacks have been called.  Which was an error in #10543.